package.json

```diff
@@ -8,7 +8,9 @@
     "build": "webpack --env.environment=production --config webpack.config.js",
     "dev": "webpack --env.environment=development --config webpack.config.js",
     "devserver": "webpack-dev-server --open --hot --env.environment=development --config webpack.config.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "sass": "node-sass --include-path scss src/styles/sass/*.scss src/styles/sass/*.css",
```

Comment:
It would be a good idea for us to use loaders for sass

Reply:
I have used sass loaders, thank you for the feedback